.github/workflows/release.yml

```
@@ -0,0 +1,220 @@
+      version: ${{ steps.headver.outputs.version }}
+      head: ${{ steps.headver.outputs.head }}
+      is_prerelease: ${{ steps.headver.outputs.is_prerelease }}
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Make HeadVer script executable
+        run: chmod +x scripts/headver.sh
+      
+      - name: Generate HeadVer version
+        id: headver
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:3:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)